rclpy/test/test_destruction.py

```diff
@@ -20,7 +20,7 @@
 def run_catch_report_raise(func, *args, **kwargs):
     try:
         return func(*args, **kwargs)
-    except:
+    except Exception:
```

Comment:
Not necessary but recommended in [PEP 8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) and some linters will warn about it:

> When catching exceptions, mention specific exceptions whenever possible instead of using a bare `except:`